src/month.rs

```diff
@@ -186,6 +186,66 @@ impl Months {
     }
 }
 
+/// A difference in a number of months, either forwards or backwards.
```

Comment:
This doesn't seem like it necessarily belongs in this PR?

Reply:
I've moved this to https://github.com/chronotope/chrono/pull/859